pkgs/development/tools/rust/cargo-flash/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkg-config, rustfmt, libusb1 }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-flash";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "probe-rs";
+    repo = pname;
+    rev = "v" + version; 
+    sha256 = "01zyxbykzbqfy6w0cmhhq10l1qqf4aqxvh4jngj1lqba5r9d7jsc";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
+  cargoSha256 = "0fpcglng00vfx5643y2dcjik29vl2rz9vv8dx67gnpbkdrhpfxcx";
+
+  nativeBuildInputs = [ pkg-config rustfmt ];
+  buildInputs = [ libusb1.dev ];
+
+  meta = with stdenv.lib; {
+    description = "A cargo subcommand to flash ELF binaries onto ARM chips";
+    homepage = "https://github.com/probe-rs/" + pname;
```

A copy-pastable URL is probably better:
```suggestion
    homepage = "https://github.com/probe-rs/cargo-flash";
```

---

Hmmh, sure

---

said yes, didn't do it :P

---

Maybe now?